src/PatientChecking/PatientChecking/Services/ImageService.cs

```diff
@@ -0,0 +1,47 @@
+﻿using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Http;
+using Newtonsoft.Json;
+using PatientChecking.Services.Repository;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace PatientChecking.Services
+{
+    public class ImageService : IImageService
+    {
+        private readonly IHostingEnvironment _hostingEnvironment;
+
+        public ImageService(IHostingEnvironment hostingEnvironment)
+        {
+            _hostingEnvironment = hostingEnvironment;
+        }
+
+        public bool IsImageFile(IFormFile formFile)
+        {
+            var fileExtension = Path.GetExtension(formFile.FileName).ToLower();
+
+            if (fileExtension != ".jpg" && fileExtension != ".png" && fileExtension != ".jpeg")
```

Comment:
Verify file extension is accepted in this case, **don't need change the current code**. But in some case, it can be security issue: what happens if hacker can bypass this logic with an .exe file? For ex: virus.exe.jpg. They can upload a virus into our system. For future, this is a suggestion: Verify the header of images:

- https://stackoverflow.com/questions/210650/validate-image-from-file-in-c-sharp
- https://stackoverflow.com/questions/670546/determine-if-file-is-an-image

Reply:
Thank you for your advice. I have noticed this and I will apply it in the future